src/Service.Tests/SqlTests/RestApiTests/Find/FindApiTestBase.cs

```diff
@@ -127,6 +127,23 @@ public virtual async Task FindOnTableWithUniqueCharacters()
                 sqlQuery: GetQuery("FindOnTableWithUniqueCharacters"));
         }
 
+        /// <summary>
+        /// Tests the Rest Api to validate that queries work
+        /// when there is the same table name in two different
+        /// schemas. In this test we have two table both
+        /// named magazines but with one in the schema "foo" and
+        /// the other in the schema "bar".
+        /// </summary>
+        [TestMethod]
+        public virtual async Task FindOnTableWithNamingCollision()
+        {
+            await SetupAndRunRestApiTest(
```

Comment:
Do you  need to verify a request against the second entity? and ensure results are as expected from expected database type?

Reply:
I think just what we are doing should be fine since this demonstrates that we correctly navigate the collision and get the expected results from the request on a tablename with a collision.